docs/concepts/configuration.md

```diff
@@ -82,6 +119,8 @@ The most common way of supplying them can be implemented by using Angular CLI en
 
 An example for this kind of settings are breakpoint settings for the different device classes of the application touch points.
 
+Another way would be the usage of `TransferState` to use environment variables of the deployment for the configuration of the application.
```

Comment:
"Another way" refers to the sentence before the previous. Please re-add context. i.e. "Deployment settings can also be ..."